packages/vat-data/src/exo-utils.js

```diff
@@ -106,7 +106,7 @@ export const makeExoUtils = VatData => {
     });
   harden(defineVirtualExoClass);
 
-  // TODO interfaceGuard type https://github.com/Agoric/agoric-sdk/issues/6206
+  // TODO interfaceGuardKit type https://github.com/Agoric/agoric-sdk/issues/6206
```

Comment:
done?

(same comment for other two)

Reply:
Good catch! Fixed a bunch of these. Thanks.